src/core/sys/netbsd/dlfcn.d

```diff
@@ -0,0 +1,114 @@
+/**
+ * D header file for NetBSD.
+ *
+ * Copyright: Copyright Martin Nowak 2012.
+ * License:   $(WEB www.boost.org/LICENSE_1_0.txt, Boost License 1.0).
+ * Authors:   Martin Nowak
+ *
+ * http://cvsweb.netbsd.org/bsdweb.cgi/~checkout~/src/include/dlfcn.h
+ */
+module core.sys.netbsd.dlfcn;
+
+public import core.sys.posix.dlfcn;
+
+version (NetBSD):
+extern (C):
+nothrow:
```

Comment:
FreeBSD file has `@nogc` attribute here, missing in yours.  Why not add that attribute to all the header files?